test/e2e/specs/specs-playwright/wp-woocommerce__landing-page.ts

```diff
@@ -0,0 +1,63 @@
+import {
+	DataHelper,
+	SidebarComponent,
+	TestAccount,
+	WoocommerceLandingPage,
+} from '@automattic/calypso-e2e';
+import { Page, Browser } from 'playwright';
+
+declare const browser: Browser;
+
+describe( DataHelper.createSuiteTitle( 'WooCommerce Landing Page' ), function () {
+	let page: Page;
+
+	describe( 'Learn more button opens a support article', function () {
+		let landingPage: WoocommerceLandingPage;
+
+		beforeAll( async () => {
+			page = await browser.newPage();
+
+			const testAccount = new TestAccount( 'defaultUser' );
+			await testAccount.authenticate( page );
+		} );
+
+		afterAll( async () => {
+			await page.close();
+		} );
```

Comment:
You actually don't need these `afterAll` calls! This is kind of confusing and we just changed this recently, so sorry for the confusion, but we handle shutting down all the new pages made by the browser holistically at the end :). So you should be able to just safely delete these!

Reply:
Thanks! Will remove!